projects/aca-content/src/lib/components/search/search-libraries-results/search-libraries-results.component.html

```diff
@@ -1,6 +1,7 @@
 <aca-page-layout>
   <aca-page-layout-header>
-    <adf-breadcrumb root="APP.BROWSE.SEARCH_LIBRARIES.TITLE"> </adf-breadcrumb>
```

Comment:
can you please also check if this translation resource used anywhere? if not, you can remove the string from the en.json file

Reply:
I have removed string from en.json file.